src/lib/services/globus-connect-server/service/collections.ts

```diff
@@ -62,3 +62,33 @@ export const get = function (
     query?: operations["getCollection"]["parameters"]["query"];
   }
 >;
+
+/**
+ * @see https://docs.globus.org/globus-connect-server/v5.4/api/openapi_Collections/#getCollection
```

Comment:
Bad link, I think it's this one: `https://docs.globus.org/globus-connect-server/v5.4/api/openapi_Collections/#deleteCollection`